app/web-app/prisma/migrations/0_init/migration.sql

```diff
@@ -1,6 +1,7 @@
 -- CreateTable
 CREATE TABLE "FundingVault" (
     "id" SERIAL NOT NULL,
+    "vaultName" TEXT NOT NULL,
     "description" TEXT NOT NULL,
     "creatorAddress" TEXT NOT NULL,
     "fundingTokenAddress" TEXT NOT NULL,
```

Comment:
@bhavik-mangla Please do not edit this file manually or use a hard reset during migration, such as running `prisma db push`. Doing so will cause Prisma to delete all previous migrations and trigger a Drift Detected warning during subsequent migrations.